style.css

```diff
@@ -0,0 +1,105 @@
+
+ .hidden{
+    display: none;
+        }
+body{
+	font-family: 'Ubuntu', sans-serif;
+	margin: 0;
+	padding: 0;
+	}
+#leaderboard-table{
+	width: 70%;
```

Comment:
Use class for the style. Don't use id for css

Reply:
done